src/contact.html

```diff
@@ -0,0 +1,30 @@
+---
+layout: default.html
+pageTitle: Contact 
+tags: nav
+order: 5
+---
+
+<div class="site-grid-contact">
+  <div class="header-contact-wrapper">
+   <h2>Contact Us!</h2>
+   <h4>Please let us know how we can do better</h4>
+  </div>
+
+  <code>contact.html</code>
+
+  <div class="content-contact-wrapper">
+    <form action="action_page.php">
+
+      <input type="text" placeholder="First and Last Name:">
+      <input type="text" placeholder="Company:">
+      <input type="text" placeholder="Your Email:">
```

Comment:
There is a separate `type` for email addresses that has some worthwhile benefits. Learn more about valid `type` attributes for the `input` element here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types.
```suggestion
      <input type="email" placeholder="Your Email:">
```